addons/ui/fnc_initDisplayInventory.sqf

```diff
@@ -0,0 +1,275 @@
+            };
+        };
+
+        [ctrlParent _control, _unit, _classname, _slotType] call FUNC(openItemContextMenu);
+    }];
+} forEach [
+    _uniformSlot, _vestSlot, _backpackSlot,
+    _uniformSlotBackground, _vestSlotBackground, _backpackSlotBackground,
+    _headgearSlot, _glassesSlot, _hmdSlot, _binocularSlot,
+    _rifleSlot, _rifleSilencerSlot, _rifleBipodSlot, _rifleOpticSlot, _riflePointerSlot, _rifleMagazine2Slot, _rifleMagazineSlot,
+    _launcherSlot, _launcherSilencerSlot, _launcherBipodSlot, _launcherOpticSlot, _launcherPointerSlot, _launcherMagazineSlot,
+    _pistolSlot, _pistolSilencerSlot, _pistolBipodSlot, _pistolOpticSlot, _pistolPointerSlot, _pistolMagazineSlot,
+    _mapSlot, _gpsSlot, _radioSlot, _compassSlot, _watchSlot
+];
+
+// Containers
+private _groundItems = _display displayCtrl IDC_FG_GROUND_ITEMS;
+private _containerItems = _display displayCtrl IDC_FG_CHOSEN_CONTAINER;
+private _uniformItems = _display displayCtrl IDC_FG_UNIFORM_CONTAINER;
+private _vestItems = _display displayCtrl IDC_FG_VEST_CONTAINER;
+private _backpackItems = _display displayCtrl IDC_FG_BACKPACK_CONTAINER;
+
+_groundItems setVariable [QGVAR(containerType), "GROUND"];
+_containerItems setVariable [QGVAR(containerType), "CARGO"];
+_uniformItems setVariable [QGVAR(containerType), "UNIFORM_CONTAINER"];
+_vestItems setVariable [QGVAR(containerType), "VEST_CONTAINER"];
+_backpackItems setVariable [QGVAR(containerType), "BACKPACK_CONTAINER"];
+
+{
+    _x ctrlAddEventHandler ["lbDblClick", {
+        params ["_control", "_index"];
+        private _unit = call CBA_fnc_currentUnit;
+
+        private _container = objNull;
+        private _containerType = _control getVariable QGVAR(containerType);
+        switch _containerType do {
+            case "GROUND": {
+                _container = GVAR(CurrentGroundItemHolder);
+            };
+            case "CARGO": {
+                _container = GVAR(CurrentContainer);
+            };
+            case "UNIFORM_CONTAINER": {
+                _container = uniformContainer _unit;
+            };
+            case "VEST_CONTAINER": {
+                _container = vestContainer _unit;
+            };
+            case "BACKPACK_CONTAINER": {
+                _container = backpackContainer _unit;
+            };
+        };
+
+        // Reports classname, but only for magazines.
+        private _classname = _control lbData _index;
+        if (_classname isEqualTo "") then {
+            // For weapons, items and glasses, use the display name to guess the classname.
+            private _displayName = _control lbText _index;
+
+            private _items = weaponCargo _container + itemCargo _container;
+            private _index = _items findIf {
+                _displayName isEqualTo getText (_x call CBA_fnc_getItemConfig >> "displayName");
+            };
+
+            _classname = _items param [_index, ""];
```

Comment:
todo test if counting weaponCargo, itemCargo works and comparing with _index works.